website/docs/reference/change-requests.mdx

```diff
@@ -104,8 +104,27 @@ While change requests are environment-specific, project segments are not. For th
 
 Only Admin users can bypass the change request process for project segments through API calls.
 
+## Environment-level change requests
```

Comment:
@kwasniew what do you think about calling it 'Environment-level change requests'? 'Global environment change requests' makes me think there is such a thing as a 'global environment'. but happy to change it back if you think it's better

Reply:
I like it :)